### DISCUSSION TITLE
Duplicate IDs

### DISCUSSION THREAD

**Discussion by saosebastiao** [OvertureMaps/data#128] [Discussion]
The current dataset, `2024-02-15-alpha.0` seems to have duplicated IDs. It's not clear to me what these represent or why there are more than one, but it does appear as though the other attribute data in the rows is different, and thus the IDs should either be different or the rows should have their data consolidated. Either way, an id column should be able to pass a uniqueness check.

**Comment by jenningsanderson** [comment]
Fully agree, we are making improvements with each release to ensure that IDs are unique. There is a known issue where IDs are duplicated within buildings because the features are duplicated in the source, i.e, they have the exact same geometry in OSM. From a stability perspective, you might expect them to have the same ID because they _are_ the same building... but in the future, we will only deliver 1 of them in the final release.
